modules/users/data/data.yaml

```diff
@@ -24,13 +24,19 @@ groups:
     description: Access to do basic stuff on tools1
     members: [sario]
     privileges: ['ALL = (ALL) NOPASSWD: /usr/sbin/service *',
-                 'ALL = (ALL) NOPASSWD: /var/phabricator/bin/*']
+                 'ALL = (ALL) NOPASSWD: /var/phabricator/bin/*',
+                 'ALL = (ALL) NOPASSWD: /usr/bin/puppet *',
+                 'ALL = (ALL) NOPASSWD: /usr/bin/git *',
```

Comment:
Change git to only allow the one repo.

Reply:
I’m not sure it’s possible to limit a command to a specific directory